programs/create-genesis/genesis_info.hpp

```diff
@@ -42,9 +43,9 @@ struct genesis_info {
 
     struct parameters {
         stake_params stake;
+        posting_rules posting_rules;
```

Comment:
Please, insert `posting_rules` in genesis-info.json (https://github.com/GolosChain/golos.contracts/blob/master/genesis/genesis-info.json.tmpl)

Reply:
resolved in GolosChain/golos.contracts#630